src/main/java/domain/piece/Empty.java

```diff
@@ -0,0 +1,28 @@
+package domain.piece;
+
+import domain.board.Position;
+import domain.rule.DefaultMoveRule;
+import java.util.List;
+
+public class Empty extends Piece {
+
+    private static final Empty INSTANCE = new Empty();
+
+    public Empty() {
```

Comment:
```suggestion
    private Empty() {
```
싱글톤 패턴을 고려하신 것이라면 private로 막아서 하나 이상 생성되지 않도록 해야하지 않을까요~?